packages/lib/src/components/NavBar/components/NavLinks.tsx

```diff
@@ -29,22 +29,18 @@ import { useTranslation } from 'next-i18next'
 
 type Props = {
   links: NavLink[]
-  user?: UserOutput
+  userRights?: string[] | null
 }
 
-export function NavLinks({ links, user }: Props): JSX.Element {
+export function NavLinks({ links, userRights }: Props): JSX.Element {
   const { t } = useTranslation()
 
   const router = useRouter()
 
   const theme = useMantineTheme()
 
   function hasRequiredRights(rights: string[]): boolean {
-    return Boolean(
-      rights.every(right =>
-        user?.role.rights.map(userRight => userRight.authority).includes(right)
-      )
-    )
+    return Boolean(rights.every(right => userRights?.includes(right)))
```

Comment:
`.every` already does return a boolean hence it needn't be explicitly coerced. But you already have written a util function `hasRequiredRights` so you can use it here right?

EDIT: Ah, I've seen that the util function is only in the `app` package. Suggestion: outsource it to the `lib` package and import it in the `app` package like other util functions.

Reply:
Very good suggestion!